tests/unit/test_logger.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import unicode_literals
+
+import traceback
+import logging
+import colorama
+
+import dvc.logger
+from dvc.exceptions import DvcException
+
+
+dvc.logger.setup()
+logger = logging.getLogger("dvc")
+formatter = dvc.logger.ColorFormatter()
+colors = {
+    "blue": colorama.Fore.BLUE,
+    "red": colorama.Fore.RED,
+    "yellow": colorama.Fore.YELLOW,
+    "nc": colorama.Fore.RESET,
+}
+
+
+class TestColorFormatter:
+    def test_debug(self, caplog):
+        with caplog.at_level(logging.DEBUG, logger="dvc"):
+            logger.debug("message")
+
+        expected = "{blue}DEBUG{nc}: message".format(**colors)
+
+        assert expected == formatter.format(caplog.records[0])
+
+    def test_info(self, caplog):
+        logger.info("message")
+
+        assert "message" == formatter.format(caplog.records[0])
+
+    def test_warning(self, caplog):
+        logger.warning("message")
+
+        expected = "{yellow}WARNING{nc}: message".format(**colors)
+
+        assert expected == formatter.format(caplog.records[0])
+
+    def test_error(self, caplog):
+        logger.error("message")
+
+        expected = (
+            "{red}ERROR{nc}: message\n"
+            "\n"
+            "{footer}".format(footer=formatter.footer, **colors)
+        )
+
+        assert expected == formatter.format(caplog.records[0])
+
+    def test_exception(self, caplog):
+        try:
+            raise ValueError
+        except Exception:
+            logger.exception("message")
```

Comment:
Is `logger.exception()` able to parse our chained exceptions?

Reply:
Yes, it works the same as `error` for backwards compatibility, but it makes more sense semantically